### DISCUSSION TITLE
Pre-configured dashboards

### DISCUSSION THREAD

**Discussion by haranathg** [bassmaster187/TeslaLogger#782] [Q&A]
Hi, thanks much for this git repo - I was able to set this up on a docker in my synology nas.  I believe I got everything running including - http://mynas:8888/admin/dashboard.php - which gives me the charge % and miles (actually it gives me KM, even though in the settings I changed it to miles).

But when I go to graphana - http://mynas:3000/?orgId=1 - I get a blank dashboard - is there a pre-configured dashboard that could be added to this repo that we can use as a starting point - for those not too familiar with graphana.

Thanks much!

**Comment by bassmaster187** [comment]
we can setup a start dashboard or what are you looking for? A guide how to use grafana?

https://grafana.com/docs/grafana/latest/administration/provisioning/
